suite/tests/CMakeLists.txt

```diff
@@ -2305,7 +2305,14 @@ if (CLIENT_INTERFACE)
                 "-thread_private -prof_pcs -opt_cleancall 0" "")
             endif (NOT X64)
           endif (UNIX AND NOT ARM)
-        endif ()
+        elseif (sample STREQUAL "memval_simple")
+          # i#2449: only witnessed crash on 32-bit linux
```

Comment:
Unless it's extra work wouldn't we want to run the test for as many platforms as possible?  Porting it to ARM is work but x86_64 should "just work"?